CMakeLists.txt

```diff
@@ -16,6 +16,7 @@ pkg_check_modules(DLT REQUIRED automotive-dlt-c++)
 add_subdirectory( deps/audio/common )
 
 string(REPLACE " -O3" " -O2" GPTP_CXX_FLAGS "${CMAKE_CXX_FLAGS}")
+string(REPLACE " -Werror" "" GPTP_CXX_FLAGS "${CMAKE_CXX_FLAGS}")
```

Comment:
This is unrelated to commit message - is it a leftover?

Reply:
yes please disregard I will resubmit.